thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -768,6 +770,7 @@ public Response replayAnomalyFunctionByFunctionId(@PathParam("functionId") long
     DateTime replayStart = ISODateTimeFormat.dateTimeParser().parseDateTime(replayStartTimeIso);
     DateTime replayEnd = ISODateTimeFormat.dateTimeParser().parseDateTime(replayEndTimeIso);
     FunctionReplayRunnable functionReplayRunnable = new FunctionReplayRunnable(detectionJobScheduler, anomalyFunctionDAO, mergedAnomalyResultDAO, rawAnomalyResultDAO, new HashMap<String, String>(), id, replayStart, replayEnd, false);
+    functionReplayRunnable.setForceBackfill(false);
```

Comment:
Set forceBackfill to true would ignore any existing anomalies in the interval to be backfilled; otherwise, the system would abort backfilling to prevent unwanted backfilling.

I don't think changing this forceBackfilling would achieve what you want. You could make backfill function takes as input a notification flag and set notified variable to that flag. (Currently, the flag is always true.)